CONTRIBUTING.md

```diff
@@ -343,11 +342,11 @@ index 791a7da1f4..a5db13b414 100644
      "@babel/runtime": "^7.4.4",
      "@mui/styled-engine": "^5.0.0-alpha.16",
 -    "@mui/material": "^5.0.0-alpha.15",
-+    "@mui/material": "https://pkg.csb.dev/mui/material-ui/commit/371c952b/@mui/material",
++    "@mui/material": "https://pkg.pr.new/mui/material-ui/@mui/material@b0f26aa",
```

Comment:
I'm going a bit off topic. What's the rationale for not using [`--compact`](https://github.com/stackblitz-labs/pkg.pr.new) option? I was thinking of killing the option in https://github.com/mui/mui-public/blob/cf68446433d3e1fcb7a2be55e7fdf866835c36b6/.github/workflows/ci-base.yml#L10. 

[Vite uses](https://github.com/vitejs/vite/blob/e899bc7c73a27cdf327875e5d696c50d396a7fc2/.github/workflows/preview-release.yml#L38) it, [Rolldown too](https://github.com/rolldown/rolldown/blob/7295247ac233396bd4a0f4b2cbe73537b4f0414e/.github/workflows/publish-to-pkg.pr.new.yml#L140), but with our npm package maze, showing the path to the source might be simpler.